src/ripple/proto/org/xrpl/rpc/v1/ledger.proto

```diff
@@ -25,16 +25,6 @@ message LedgerSpecifier
     }
 }
 
-// Next field: 3
-message LedgerRange
```

Comment:
Do we use this in Clio or reporting mode, or only get the LedgerRange from the `ledger` subscription stream?

Reply:
We don't use this in clio or reporting mode